exporter/templates/cat1/r3/cat1.xml

```diff
@@ -86,16 +86,9 @@
     <author>
       <time value="{{timeNow}}"/>
       <assignedAuthor>
-        <!-- id extension="Cypress" root="2.16.840.1.113883.19.5"/ -->
         <!-- NPI -->
-        {{if .CMSCompatibility}}
-          {{range .Record.ProviderPerformances}}
-            {{range .Provider.CDAIdentifiers}}
-              {{if eq .Root "2.16.840.1.113883.4.6"}}
-                <id root="2.16.840.1.113883.4.6" extension="{{.Extension}}" />
-              {{end}}
-            {{end}}
-          {{end}}
+        {{range .Record.ProviderIdentifiersForRoot "2.16.840.1.113883.4.6" "1234567893" }}
+          <id root="2.16.840.1.113883.4.6" extension="{{.}}" />
```

Comment:
The provider IDs are allowed by the HL7 Implementation Guide, they are just only required by the CMS IG. Cypress generates NPIs for all providers, but if there isn't an NPI, this function will insert "1234567893" instead, which is a valid NPI.